modules/upcloud.py

```diff
@@ -183,8 +186,8 @@ def find_server(self, uuid, hostname):
             try:
                 server = self.manager.get_server(uuid)
                 return server
-            except Exception, e:
-                pass # no server found
+            except Exception:
```

Comment:
Exceptions need to be refactored like this since the old way does not work in python3